src/main.rs

```diff
@@ -37,6 +37,9 @@ struct Args {
     /// show the executable version
     #[argh(switch, short = 'v')]
     version: bool,
+    /// show hints on success
+    #[argh(switch)]
+    success_hints: bool,
```

Comment:
This should be added to `WatchArgs` instead, because while this works, you'd need to type `rustlings --success-hints watch` to make it work, `rustlings watch --success-hints` is way more ergonomic.

Reply:
Thank you for your feedback, I added it to WatchArgs instead of Args. It works as you said. Now previously I put this option for the verify command too, because I modified the parameters of the function verify, but here I put false for the success_hints parameter when the command verify is called.